data-pipeline/src/pipeline/config.py

```diff
@@ -1,20 +1,45 @@
 income_category_map = {
-    "academies": [
-        "DFE/EFA Revenue grants (includes Coronavirus Government Funding",
-        "of which: Coronavirus Government Funding",
-        "SEN funding",
-        "Other DfE/EFA Revenue Grants",
-        "Other income - LA & other Government grants",
-        "Government source, non-grant",
-        "Academies",
-        "Non-Government",
-        "All income from facilities and services",
-        "Income from catering",
-        "Receipts from supply teacher insurance claims",
-        "Donations and/or voluntary funds",
-        "Other self-generated income",
-        "Investment income",
-    ]
+    "academies": {
+        # "DFE/EFA Revenue grants (includes Coronavirus Government Funding": "Income_DFE revenue grants",
+        # "of which: Coronavirus Government Funding": "",
+        "SEN funding": "Income_SEN funding",
+        "Other DfE/EFA Revenue Grants": "Income_Other DFE grants",
+        "Other income - LA & other Government grants": "Income_Other grants",
+        "Government source, non-grant": "Income_Government source",
+        "Academies": "Income_Academies",
+        "Non-Government": "Income_Non government",
+        "All income from facilities and services": "Income_Facilities and services",
+        "Income from catering": "Income_Catering services",
+        "Receipts from supply teacher insurance claims": "Income_Receipts supply teacher insurance",
+        "Donations and/or voluntary funds": "Income_Donations and voluntary funds",
+        "Other self-generated income": "Income_Other self-generated income",
+        "Investment income": "Income_Investment income",
+        "Direct Grants": "Income_Direct grants",
+        "Community Grants": "Income_Community grants",
+        "Total Grant Funding": "Income_Total grant funding",
+        "Total Self Generated Funding": "Income_Total self generated funding",
+    },
+    "maintained_schools": {
+        # "I01  Funds delegated by the LA": "",
+        # "I02  Funding for 6th form students": "",
+        # "I03  SEN funding": "Income_SEN funding",
+        # "I04  Funding for minority ethnic pupils": "",
+        # "I05  Pupil Premium": "",
```

Comment:
What is the reason for not including these? I think we do have mappings for them

Reply:
Not directly at the minute see this ticket - https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_workitems/edit/212289